install.ps1

```diff
@@ -224,6 +227,12 @@ function Invoke-WinsInstaller {
             $env:CATTLE_AGENT_LOGLEVEL = $env:CATTLE_AGENT_LOGLEVEL.ToLower()
         }
 
+        if ((-Not $env:STRICT_VERIFY) -and ($StrictTlsVerification)) {
+            $env:STRICT_VERIFY = "true"
+        } elseif (-Not $env:STRICT_VERIFY){
+            $env:STRICT_VERIFY = "false"
+        }
```

Comment:
Does it cover the case where the env var STRICT_VERIFY has been set to false and we want to change it to true by providing StrictTlsVerification?